src/pages/CourseList/CourseCard/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import Card from 'react-bootstrap/Card';
+import Video from './Video';
+import Content from './Content';
+import '../styles.scss';
+import { ICourse } from '../../../interfaces/Course.interface';
+
+interface ICourseCardProps {
+  data: ICourse;
+}
+
+const CourseCard = ({ data }: ICourseCardProps) => {
+  const { meta } = data;
+  const [isHovering, setIsHovering] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isHovering, setIsHovering] = useState(false);
```
unnecessary overtyping